cmd/binapi-generator/main.go

```diff
@@ -18,53 +18,64 @@ import (
 	"flag"
 	"fmt"
 	"os"
-	"path/filepath"
-	"strings"
-	"unicode"
 
 	"github.com/sirupsen/logrus"
 
 	"go.fd.io/govpp/binapigen"
 	"go.fd.io/govpp/binapigen/vppapi"
 	"go.fd.io/govpp/version"
+	_ "go.fd.io/govpp/binapigen/plugins"
 )
 
 func init() {
 	flag.Usage = func() {
-		fmt.Fprintf(os.Stderr, "USAGE\n")
-		fmt.Fprintf(os.Stderr, "  Parse API_FILES and generate Go bindings\n")
-		fmt.Fprintf(os.Stderr, "  Provide API_FILES by file name, or with full path including extension.\n")
-		fmt.Fprintf(os.Stderr, "  %s [OPTION] API_FILES\n\n", os.Args[0])
+		fmt.Fprintf(os.Stderr, "GoVPP api bindings generation CLI\n")
+		fmt.Fprintf(os.Stderr, "---------------------------------\n\n")
+		fmt.Fprintf(os.Stderr, "This CLI generates VPP api Go bindings based on .api files or a VPP repository\n\n")
+		fmt.Fprintf(os.Stderr, "Usage: ./binapigen [OPTION]\n")
 		fmt.Fprintf(os.Stderr, "OPTIONS\n")
 		flag.PrintDefaults()
 		fmt.Fprintf(os.Stderr, "\nEXAMPLES:\n")
-		fmt.Fprintf(os.Stderr, "  %s \\\n", os.Args[0])
-		fmt.Fprintf(os.Stderr, "    --input-dir=$VPP/build-root/install-vpp-native/vpp/share/vpp/api/ \\\n")
-		fmt.Fprintf(os.Stderr, "    --output-dir=~/output \\\n")
-		fmt.Fprintf(os.Stderr, "    interface ip\n")
-		fmt.Fprintf(os.Stderr, "  Assuming --input-dir contains interface.api.json & ip.api.json\n")
+		fmt.Fprintf(os.Stderr, "  Generate bindings from .api files :\n")
+		fmt.Fprintf(os.Stderr, "   ./binapigen --api /usr/share/vpp/api --output ./myapp/binding --filter interface,ip\n")
+		fmt.Fprintf(os.Stderr, "  Generate bindings from the VPP repository :\n")
+		fmt.Fprintf(os.Stderr, "   ./binapigen --vpp ~/vpp --output ./myapp/binding\n")
 	}
 }
 
 func printErrorAndExit(msg string) {
-	fmt.Fprintf(os.Stderr, "Error: %s\n\n", msg)
+	fmt.Fprintf(os.Stderr, "\033[31mError:\n%s\033[0m\n\n", msg)
```

Comment:
Using some ANSI color library like [`github.com/fatih/color`](https://pkg.go.dev/github.com/fatih/color) will take care of omitting color escapes if the output is not a terminal, which will avoid _uglyfying_ the log output if it goes to a file or simply does no run in the shell.